firmware/user/modes/mode_test.c

```diff
@@ -370,7 +382,7 @@ static void ICACHE_FLASH_ATTR testLedFunc(void* arg __attribute__((unused)))
 static void ICACHE_FLASH_ATTR testAnimateSprite(void* arg __attribute__((unused)))
 {
     // test.rotation = (test.rotation + 90) % 360;
-    test.rotation = (test.rotation + 3) % 360;
+    test.rotation = (test.rotation + test.rotationInc + 360) % 360;
```

Comment:
Adding 360 to a number you mod 360 doesn't have any effect.

Reply:
I wasn't sure if you handled negative angles so that insured rotation was non-negative and less than 360.